CONTRIBUTING.md

```diff
@@ -173,64 +178,64 @@ will trigger notifications to tens, hundreds, potentially thousands of people. A
 
 Make sure to test your own code locally and/or in CI before submitting a pull request to avoid numerous updates
 afterwards to fix CI (remember, those changes will trigger extra notifications that could be avoided by following this
 simple rule). Make sure to add new test cases if applicable.
 
-Pull request should be accompanied by reasonable description. If the change is a trivial typo, description can be
-avoided completely, but if changes are non-trivial, make sure to explain what changes do, why, what were the
-alternatives and so on, giving reviewer proper context into changes proposed. If there is a relevant issue or another
-pull request then link it, if there is an issue that is resolved by proposed changes or PR that becomes obsolete, use
-corresponding keywords as described in
-[documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
-
-Pull request should be ready for review, avoid drafts that are frequently updated, in most cases you can share WIP
-changes with someone just by sharing a branch diff. Drafts have a narrow and rare use case, use them as last resort,
-otherwise they become a huge source of distraction due to number of notifications they generated. In most cases you
-either want changes to be reviewed and merged, in which case regular pull request should be created, or you don't want
-pull request at all.
-
-Once PR is created, a rule of thumb is to avoid force pushes since it is not always easy to understand what has changed
-in this case and forces maintainers to re-review pull request from scratch. Prefer meaningful commits on top instead.
+Pull requests should be accompanied by reasonable description. If the change is a trivial typo, the description can be
+empty, but if changes are non-trivial, make sure to explain what the changes do, why, what were the alternatives and so on,
+giving the reviewer proper context into the proposed changes. If there is a relevant issue or another pull request then
+link it, if there is an issue that is resolved by proposed changes or PR that becomes obsolete, use
+[GitHub linking keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
+
+Pull requests should be ready for review, avoid opening drafts that are frequently updated, in most cases you can share
+WIP changes with someone just by sharing a branch diff. Drafts have a narrow and rare use case, use them as last resort,
+otherwise they become a huge source of distraction due to number of notifications they generate. In most cases you
+either want changes to be reviewed and merged, in which case a regular pull request should be created, or you don't want
+a pull request at all. If an open pull request stops being ready for review or merging, close it or put it in draft.
+Pull requests should not stay in draft longer than a week, they can be closed now, and re-opened later when ready.
```

Comment:
This is something @vedhavyas and I talked about recently.

Reply:
I split those changes into two different parts of the paragraph here:
https://github.com/autonomys/subspace/pull/3569#pullrequestreview-2911512855